### DISCUSSION TITLE
What better way to edit information in the model (image link, name in the database)?

### DISCUSSION THREAD

**Discussion by ewertonrfd** [adonisjs/core#3715] [Help] [answered]
I have the following field in the model

```
@column()
public logo: string | null;
```

I need that instead of just showing the image name
Show the full link:
"https://mysite.com.br/img/" + logo
But I don't want to be storing this in the database(the prefix, path)

I tried something like:
```
@column({
  serialize: async (value: string | null) => {
    return value ? await Drive.getUrl(value) : value;
  },
})
public logo: string | null;
```
But still not sure 😣😅

I'm reading around here in the doc
https://docs.adonisjs.com/guides/models/serializing-models#mutatingtransforming-values

Thank you for your help

**Comment by thetutlage** [comment, marked as answer]
You can use the model hooks for this

**Reply by ewertonrfd** [reply]
Thanks for the tip

I hadn't used hooks like that, yet (except for the password, which I hadn't realized you could do something like that, in the query)
